_posts/2021-04-27-Optimising-ux-for-software.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Optimising UX for Software"
+date: 2021-04-27
+tags: ux devops design
+author: Lucy Sharp
+image: /img/User_Journey_Map.jpeg
+---
+
+My baptism into the cult of Mechanical Rock kicked off with a sword fight (legally, I can't say more) and it got me thinking about the relationship between UX and DevOps. Particularly, how DevOps drives the evolution of UX design.
```

Comment:
cult!?

Reply:
When I first joined I remember you describing MR as a cult

Reply:
was I drunk?

Reply:
Yeh